### DISCUSSION TITLE
Matrix/IRC channel?

### DISCUSSION THREAD

**Discussion by DylanVanAssche** [fwupd/fwupd#4320] [Q&A] [answered]
Hi!

I got already several requests from some community members in the Linux Mobile community asking me where they could join the Matrix/IRC channel of fwupd. AFAIK, this doesn't exist yet. Is there any interest into such a thing? I can imagine that this isn't really applicable to most hardware vendors, but since we are now supporting community firmware as well, it might be interesting to discuss this.

**Comment by hughsie** [comment, marked as answer]
There's #fwupd on Libera -- does that count? :)

**Reply by DylanVanAssche** [reply]
Oh yes certainly! All Matrix users can join that one through the IRC bridge :) I just joined.
Would it be useful to document this somewhere?

**Reply by hughsie** [reply]
Adding it to the toplevel Readme.md might be a good plan. Got any good prose?

**Reply by DylanVanAssche** [reply]
Agreed! Give me a sec :)
